### ISSUE TITLE
How to install from source code ?

### ISSUE THREAD

**Issue by cyril-we** [oguzhaninan/Stacer#70]
I wanted to install from source but I wasn't able to do it. I followed these instructions : 

`git clone https://github.com/oguzhaninan/Stacer.git`

`cd Stacer`

`npm install && npm run build && npm start`

**Comment by oguzhaninan** [comment]
Hi,
What is the mistake you get.?

**Comment by liberodark** [comment]
is correctly what is the worry ?

**Comment by cyril-we** [comment]
The error is : 

`gyp: binding.gyp not found (cwd: /home/cyril/Stacer/node_modules/fcopy-pre-bundled) while trying to load binding.gyp`

**Comment by oguzhaninan** [comment]
What is your operating system and release ?

**Comment by cyril-we** [comment]
Linux Mint 18.1 with 4.10.0-22-generic kernel

**Comment by liberodark** [comment]
For me wich commit 216 is the same is not work on manjaro or Solus or Ubuntu

**Comment by liberodark** [comment]
I have done some wholesale testing it is a version problem for me

**Comment by psukys** [comment]
Seems like a problem with electron-builder. Reference: [electron-builder/issues/1651](https://github.com/electron-userland/electron-builder/issues/1651)

**Comment by sheepdestroyer** [comment]
same pb here with fedora 26

**Comment by joeykrim** [comment]
On Ubuntu 16.04.2 - None of the above steps for npm resolved the issue.
The above comment https://github.com/oguzhaninan/Stacer/issues/70#issuecomment-310643442 links to an issue [Cannot install fcopy-pre-bundled ](https://github.com/electron-userland/electron-builder/issues/1651) in electron-builder, where there's a suggestion to swap npm for yarn, which resolved this issue on Ubuntu 16.04.2.
Steps:
Install yarn: https://yarnpkg.com/lang/en/docs/install/
```
curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
```
Now, instead of the `npm install && npm run build && npm start` command, run `yarn`

**Comment by mnsekh111** [comment]
@sheepdestroyer Updating node to version 8+ and adding build essential solved the issue for me.
`curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -`
`sudo apt-get install -y nodejs`
`sudo apt-get install -y build-essential`

Also ty deleting node_modules/ and reinstalling again . For a few others restarting the maching solved the issue.
